# See Dynamixel documentation for details
float64 torque
---
string result
float64 position
